assertion/assertion.mbt

```diff
@@ -140,20 +140,6 @@ pub fn assert_is[T : Debug](
   }
 }
 
-test "assert_is.is" {
-  let x = 1
-  let s = "x:\(x)"
```

Comment:
we should fix `assert_is` here instead

Reply:
My concern is that will put testing only type definition `T` in the `assertion.mbt`, which decrease source code readability.